src/commands.js

```diff
@@ -114,47 +116,45 @@ export const help = msg => {
 };
 
 // Send details of a given command
-export const helpSpecific = msg => {
-  const { content, channel } = msg;
-  const userRoles = msg.member.roles;
-  const command = content.split(' ')[1];
+export const helpSpecific = (msg, command) => {
+  // const userRoles = msg.member.roles;
```

Comment:
Remove this comment and again I'd prefer if you used `const { content, channel } = msg;`